### ISSUE TITLE
Peer review feedback

### ISSUE THREAD

**Issue by AnitaLi-0371** [UBC-MDS/DSCI_522_Group_18#38]
Hi all,

My name is Anita Li, one of the reviewers for this week.

#### Documentations
In general, `README` file and final report look great. README file includes all required sections, and provide enough background and introduction to the questions. Report is easy to follow as well. I really like you put the summary at the beginning. Just a few minor formatting related improvements that you may consider:
- In README, you can probably use `>50K` in code format instead of "", which I find the former one is more readable. You could try the same in the final report.
- In final report, it may loo nicer to round numbers with less digital numbers in `Table 1` and `Table 3`.
- Figures with both titles and figure caption seem redundant to me, but it is just my own opinions. Another thing is y-axis titles, which seems misleading to me because you have done some scaling on those attributes, for example, age and working hours cannot be negative in real world. I would suggest change the axis title to something like `scaled age` etc.

#### Code
The codes are concise and easy to follow. They all run smoothly and I have no issues with them. 

#### Analysis
The analysis follows what we learn in machine learning course, and results are presented in tables which are easy to understand. Recommendations also follow the analysis part pretty well.

#### Communication
Commits are meaning and you used issues to communicate with members and keep tracking on the project, which are great. One thing I noticed is about pull request. As mention by other reviewers as well, it probably be better to get group members review it before merge. 

In summary, I think this analysis is great. Topic is interesting and report is easy to understand.

**Comment by AishwaryaGopal12** [comment]
@AnitaLi-0371 Thank you so much for the feedback. 
@yhchen20 @fei-chang I changed the figure titles and captions and mentioned in the figure explanations that the variables are scaled.
